README.md

```diff
@@ -1,2 +1,13 @@
 # cli-games
-Games in the terminal
+## About this project:
+This repository contains a collection of cli (command line iterface games). Please feel free to have a look through the code of each game (usually just a single file of code) after playing, this will help you understand the logic that goes behind making such games.
+## How to play ?
+1. Clone this repo
+2. Install required environment (node for js and python interpreter for python)
+3. Execute the game file inside terminal (like node game.js or python game.py)
+4. Enjoy
+5. Contribute!
+## Contribute :
+We welcome your contributions! You can add a new game by assigning a fresh directory to it, containing its code. For detailed guidelines refer CONTRIBUTION_GUIDELINES.md
```

Comment:
Since we don't have a CONTRIBUTION_GUIDELINES file you may remove this from here. But you are encouraged to create a guidelines file and add a link of it here